src/components/VContentReport/VContentReportButton.vue

```diff
@@ -1,12 +1,12 @@
 <template>
   <VButton
     variant="plain"
-    class="report-button font-semibold text-dark-charcoal"
+    class="report-button font-semibold text-dark-charcoal-70"
   >
-    <span class="md:hidden">{{
+    <span class="text-sr md:text-base md:hidden">{{
       $t('media-details.content-report.short')
     }}</span>
-    <span class="hidden md:inline">{{
+    <span class="text-sr md:text-base hidden md:inline">{{
```

Comment:
Style updates to the reporting button so it matches the mockups:

![CleanShot 2022-04-14 at 16 16 24@2x](https://user-images.githubusercontent.com/6351754/163468696-c5bc4863-75aa-4179-8b7c-2d597098587c.png)